12_listcomprehension_lab/02.py

```diff
@@ -1,6 +1,14 @@
-"""
-Write a python program that takes two words
-as sys.argv and prints only the letters
-common to both
-"""
-
+import operator
+
+lst1 = []
+lst2 = []
+lst3 = []
+ascii = range(97, 123)
+for i in ascii:
+     lst1.append([ x for x in chr(i)])
+     lst2.append([ x for x in chr(i)])
+     lst3.append([ x for x in chr(i)])
+
+sequence = [ (x + y + z) for x in lst1 for y in lst2 for z in lst3 ]
+
+print sequence
```

Comment:
import operator

ascii_for_lower_case_letters = range(97, 123)

char = [chr(i) for i in ascii_for_lower_case_letters]

sequence = [ (x+y+z) for x in char for y in char for z in char ]

print sequence